### PR TITLE
add option to switch off ssl-verification in requests

### PR DESCRIPTION
**Pull request by stefina**

### CODE REVIEW

**Comment by TkTech** [comment]
As someone who has to work behind a government-crippled firewall, :+1:

File: datapusher/jobs.py [hunk 1]
```diff
@@ -28,6 +29,10 @@
 
 MAX_CONTENT_LENGTH = web.app.config.get('MAX_CONTENT_LENGTH') or 10485760
 DOWNLOAD_TIMEOUT = 30
+SSL_VERIFY = asbool(web.app.config.get('ckan.datapusher.check_ssl_verify'))
```
Comment (amercader) [inline datapusher/jobs.py]: As this is a Datapusher config option I'd suggest following the convention on `datapusher_settings.py` and use `SSL_VERIFY` as the config option name. Also to save us from another external requirement I think the following should be robust enough, considering that we want this to definitely be turned off by default:

```python
if web.app.config.get('SSL_VERIFY') in ['False', 'FALSE', '0']:
    SSL_VERIFY = False
else: 
    SSL_VERIFY = True
```

**Comment by stefina** [comment]
@amercader Thank you for your feedback! I changed my PR according to your suggestions. Let me know if there are still changes required.

**Comment by amercader** [comment]
All good, thanks @stefina!

**Pull request merged**